publications-2018.bib

```diff
@@ -677,4 +677,31 @@ @Article{Chandrashekar2018a
   url       = {https://rdcu.be/725Z},
 }
 
+
+@article{doi:10.1093/gji/ggy466,
+  author = {Perry-Houts, Jonathan and Karlstrom, Leif},
+  title = {Anisotropic viscosity and time-evolving lithospheric instabilities due to aligned igneous intrusions},
+  journal = {Geophysical Journal International},
+  volume = {},
+  number = {},
+  pages = {ggy466},
+  year = {2018},
+  doi = {10.1093/gji/ggy466},
+  URL = {http://dx.doi.org/10.1093/gji/ggy466},
```

Comment:
If the URL points to the same page as the DOI, then this is redundant (since we create a link through the DOI itself). I'd like to leave the URL open to link straight to a PDF (be it preprint, thesis, whatever). I started making this change in the few entries I've edited so far. What do you think? If you agree then we should add this point to the guidelines in the README file.

Reply:
Well, sort of. They generate a clickable link that takes you straight to the page pointed to by `doi.org/<no>`. Take a look at the entry generated for `The deal.II Library, Version 9.0`, that only lists a DOI number. If we added the url `https://www.degruyter.com/view/j/jnma.just-accepted/jnma-2018-0054/jnma-2018-0054.xml` then this would be totally redundant. It'd be nicer to add a URL to the preprint PDF, IMO.

Reply:
I was talking about the rendered HTML. 

Well, guess I don't really mind if we keep these or not. It looks like I just have a different way of thinking about this than you do.